viz.yaml

```diff
@@ -492,11 +494,27 @@ publish:
                   "map_states", "map_counties", "watermark_utils", "map_circles", "dropdown_selector", "modernizr"]
       embed: ["map_section"]
   -
-    id: thumb-facebook
-    location: images/WU_stories_static_thumbs.png
+    id: thumb_facebook
     publisher: thumbnail
+    location: images/thumb_facebook.png
+    relpath: images/thumb_facebook.png
     mimetype: image/png
-    thumbType: ["facebook","twitter"]
     title: "Facebook Thumbnail"
     alttext: "Facebook Thumbnail"
+  -
+    id: thumb_twitter
+    publisher: thumbnail
+    location: images/thumb_twitter.png
```

Comment:
_if_ you go w/ my suggestion to have these generated as part of the `vizmake()` process, I think the `images/thumb_{}.png` should be gitignored.